Mathlib/Analysis/Meromorphic/Order.lean

```diff
@@ -411,28 +506,37 @@ theorem order_add (hf₁ : MeromorphicAt f₁ x) (hf₂ : MeromorphicAt f₂ x)
   have : f₁ + f₂ =ᶠ[𝓝[≠] x] ((· - x) ^ n) • g := by
     filter_upwards [h₃g₁, h₃g₂, self_mem_nhdsWithin]
     simp_all [g, ← smul_assoc, ← zpow_add', sub_ne_zero]
-  have t₀ : MeromorphicAt ((·  - x) ^ n) x := by fun_prop
-  have t₁ : t₀.order = n := t₀.order_eq_int_iff.2 ⟨1, analyticAt_const, by simp⟩
-  rw [(hf₁.add hf₂).order_congr this, t₀.order_smul h₁g.meromorphicAt, t₁]
-  exact le_add_of_nonneg_right h₁g.meromorphicAt_order_nonneg
+  have t₀ : MeromorphicAt ((· - x) ^ n) x := by fun_prop
+  have t₁ : meromorphicOrderAt ((· - x) ^ n) x = n :=
+    (meromorphicOrderAt_eq_int_iff t₀).2 ⟨1, analyticAt_const, by simp⟩
+  rw [meromorphicOrderAt_congr this, meromorphicOrderAt_smul t₀ h₁g.meromorphicAt, t₁]
+  exact le_add_of_nonneg_right h₁g.meromorphicOrderAt_nonneg
+
+@[deprecated (since := "205-05-21")]
+alias MeromorphicAt.order_add := meromorphicOrderAt_add
 
 /--
-Helper lemma for MeromorphicAt.order_add_of_order_ne.
+Helper lemma for meromorphicOrderAt_add_of_ne.
 -/
-lemma order_add_of_order_lt_order (hf₁ : MeromorphicAt f₁ x) (hf₂ : MeromorphicAt f₂ x)
-    (h : hf₁.order < hf₂.order) :
-    (hf₁.add hf₂).order = hf₁.order := by
+lemma meromorphicOrderAt_add_of_lt (hf₂ : MeromorphicAt f₂ x)
```

Comment:
```suggestion
lemma meromorphicOrderAt_add_eq_left_of_lt (hf₂ : MeromorphicAt f₂ x)
```
to match `analyticOrderAt_add_eq_left_of_lt`, and can we also have the right version?